_stbt/core.py

```diff
@@ -1767,7 +1772,26 @@ def teardown(self):
         # we won't be raising any errors from now on anyway:
         self._raise_in_user_thread = None
 
-    def push_sample(self, sample):
+    def on_sample(self, sample):
+        """
+        Called from `Display` for each frame.
+        """
+        # The test script can draw on the video, but this happens in a different
+        # thread.  We don't know when they're finished drawing so we just give
+        # them 0.5s instead.
+        SINK_LATENCY_SECS = 0.5
```

Comment:
This says 0.5s but the commit message says 1s.

Reply:
Done